src/Promitor.Agents.Scraper/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -224,9 +225,9 @@ private static void AddAtlassianStatuspageMetricSink(string pageId, IServiceColl
 
         const string OpenTelemetryServiceName = "promitor-scraper";
 
-        private static void AddOpenTelemetryCollectorMetricSink(string collectorUri, string agentVersion, IServiceCollection services, Table metricSinkAsciiTable)
+        private static void AddOpenTelemetryCollectorMetricSink(Promitor.Integrations.Sinks.OpenTelemetry.Configuration.OpenTelemetryCollectorSinkConfiguration otelConfiguration, string agentVersion, IServiceCollection services, Table metricSinkAsciiTable)
         {
-            metricSinkAsciiTable.AddRow("OpenTelemetry Collector", $"Url: {collectorUri}.");
+            metricSinkAsciiTable.AddRow("OpenTelemetry Collector", $"Url: {otelConfiguration.CollectorUri}.");
```

Comment:
We need to include protocol here

Reply:
Added

Reply:
Can you please merge it in the same row given the row title is OpenTelemetry Collector please?